### PR TITLE
Bug zap

### PR DESCRIPTION
**Pull request by kateharwood**

### CODE REVIEW

File: components/BugZap/BugZapGame.js [hunk 1]
```diff
@@ -132,7 +132,8 @@ class BugZapGame extends React.Component {
     this.fps = 20;
     this.setState({
       characterAnimationIndex: [1,2,3,4,5,6,7],
-    });    // reset characters to default state
+      splashAnimationIndex: [3,4,5],
```
Comment (micahrye) [inline components/BugZap/BugZapGame.js:135]: Why are aren't you using the character animationIndex function? and animation stuff? ie. character.animationIndex('SPLASH').

Comment (micahrye) [reply components/BugZap/BugZapGame.js:135]: see MatchColor for example

**Review by micahrye — CHANGES_REQUESTED** [review]

File: components/BugZap/BugZapGame.js (hunk 1 above)
Comment (kateharwood) [reply components/BugZap/BugZapGame.js:135]: changed

**Comment by micahrye** [comment]
I merged animate with new media and master. Now merging this PR.

**Pull request merged**